docs/source/tutorial/cli/index.rst

```diff
@@ -42,18 +44,19 @@ For more details on each command, see the corresponding section below:
 * :ref:`train`
 * :ref:`predict`
 * :ref:`convert`
+* :ref:`hpopt`
+* :ref:`fingerprint`
 
-The following features are not yet implemented, but will soon be included in a future release:
+The following features are not yet implemented, but may be included in a future release:
```

Comment:
okay! are we okay with using this language then? ("may" instead of "will soon")?

Reply:
My personal opinion is not to use "may." For something that hasn't been implemented, we can simply say "not implemented yet" or "will be implemented," since "very soon" does not provide much context about when it will become available (we know it will happen in version 2.1, which is scheduled for October). In this case, we should probably just keep it as "will be."